manage/roles/user-preferences-permissions.rst

```diff
@@ -45,6 +45,8 @@ User Preferences Permissions
 
                                        .. note:: Agents only receive ticket notifications for
                                           :doc:`groups they have “full” access to </manage/groups/access-levels>`.
+
+                                       .. hint:: Customers don't need this permission.
```

Comment:
Would prefer to combine these two notes, maybe like so?

```suggestion
                                       .. note:: Agents can only receive ticket notifications for
                                          :doc:`groups they have “full” access to </manage/groups/access-levels>`.
                                          Customers can’t receive ticket notifications at all.
```

Reply:
Thanks for your input, will be address with commit 229e5a8c55fcf709e0eadb232d165c1959a47a63